admin_guide/custom_resource_definitions.adoc

```diff
@@ -50,120 +63,192 @@ spec:
 
-[admin-guide-custom-resource-create-obj]
-## Create Custom Objects
+[admin-guide-custom-resource-create-aggregated-cluster-role]
+== Creating cluster roles for the custom resource definition
 
-After the custom resource definition object has been created, you can create
-custom objects.
+After you create cluster-scoped CRDs, you can grant permissions to it. If you
+use the admin, edit, and view 
+xref:../../architecture/additional_concepts/authorization.adoc#roles[default
+cluster roles], take advantage of cluster role aggregation for their rules.
 
-Custom objects can contain custom fields. These fields can contain arbitrary JSON.
+[IMPORTANT]
+====
+You must explicitly assign permissions to each of these roles. The roles 
+more permissions do not inherit rules from roles with fewer permissions. If you
+assign a rule to a role, you must also assign that verb to roles that have more
+permissions. For example, if you grant the "get" permission to the view role, 
+you must also grant it to the edit and admin roles. The admin or edit role is 
+usually assigned to the user that created a project through the project 
+template.
+====
 
-In the following example, the `cronSpec` and `image` custom fields are set in a
-custom object of kind `CronTab`. The kind `CronTab` comes from the `spec.kind` field of the
-custom resource definition object you created above.
+. Create a cluster role definition file for the CRD. The cluster role definition is 
+a YAML file that contains the rules that apply to each role. The {product-title}
+controller adds the rules that you specify to the default roles.
++
+.Example YAML file for a custom resource definition
 
-.Example YAML file for a Custom Object
+[source,yaml]
+----
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1 <1>
+metadata:
+  name: aggregate-cron-tabs-edit <2>
+  labels:
+    rbac.authorization.k8s.io/aggregate-to-admin: "true" <3>
+    rbac.authorization.k8s.io/aggregate-to-edit: "true" <4>
+rules:
+- apiGroups: ["stable.example.com"] <5>
+  resources: ["crontabs"] <6>
+  verbs: ["get", "list", "watch", "create", "update", "patch", "delete"] <7>
+---
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: aggregate-cron-tabs-view <2>
+  labels:
+    # Add these permissions to the "view" default role.
+    rbac.authorization.k8s.io/aggregate-to-view: "true" <8>
+rules:
+- apiGroups: ["stable.example.com"] <5>
+  resources: ["crontabs"] <6>
+  verbs: ["get", "list", "watch"] <7>
+----
++
+<1> Use the `apiextensions.k8s.io/v1beta1` API.
+<2> Specify a name for the definition.
+<3> Specify this line of code to grant permissions to the admin default role. 
+<4> Specify this line of code to grant permissions to the edit default role. 
+<5> Specify the group name of the CRD.
+<6> Specify the plural name of the CRD that these rules apply to.
+<7> Specify the xref:../../architecture/additional_concepts/authorization.adoc#action[verbs]
```

Comment:
Should mention read and write permissions.

Reply:
@enj, is this better?